### ISSUE TITLE
Add note on submodules to README?

### ISSUE THREAD

**Issue by thompson318** [ome/omero-deployment-examples#75]
It took me a little while to work out that I had to install the git submodules to get any of the examples to work. i.e. ansible-example-omero-onenode is an empty directory until you do `git submodule init` then `git submodule update`. Is it worth adding this to the README to save new users some time?

**Comment by joshmoore** [comment]
Yes, please! That makes complete sense. And sorry for the confusion.
